l10n_nl_partner_salutation/models/res_partner.py

```diff
@@ -19,15 +33,21 @@ def on_change_use_manual_salutations(self):
             self.salutation_manual = self.salutation
             self.salutation_address_manual = self.salutation_address
 
-    @api.model
-    def get_salutation_name_format(self):
-        """Return the desired name format for use in the salutation."""
-        return (
-            "${firstname or initials or ''}"
-            "${' ' if (firstname or initials) else ''}"
-            "${infix or ''}"
-            "${' ' if infix else ''}"
-            "${lastname or ''}"
+    @api.multi
+    def get_salutation_name(self):
+        """Return the name formatted for use in salutation."""
+        self.ensure_one()
+        # Use API from partner_firstname
+        # (actually method as it has been overridden in partner_name_dutch)
+        return self.with_context(
+            name_format="${firstname or initials or ''}"
```

Comment:
the idea for having `get_salutation_name_format` in its own function was modularity, so other modules can change that format as they please. Any reason to destroy this possibility?